meetups/20180502.org

```diff
@@ -0,0 +1,130 @@
+#+TITLE: Math in Emacs: The Basics (and Not-so-basics) of Calc
+#+DATE: 2 May 2018
+#+AUTHOR: with special guest Harry Schwartz
+
+* Tech Check
+*** Wi-Fi
+    :PROPERTIES:
+    :BEAMER_env: frame
+    :END:
+
+- SSID "Triplebyte Guest"
+- Password "welcome01"
```

Comment:
shouldn't we redact passwords from publicly visible documents?

Reply:
Oh good one :(